IdentityServer/v6/docs/content/quickstarts/1_client_credentials.md

```diff
@@ -4,101 +4,125 @@ date: 2020-09-10T08:22:12+02:00
 weight: 2
 ---
 
-The following quickstart provides step by step instructions for various common Duende IdentityServer scenarios. 
-These start with the absolute basics and become more complex as they progress. We recommend that you follow them in sequence.  
+Welcome to the first quickstart for IdentityServer! To see the full list of
+quickstarts, please see [Quickstarts Overview]({{< ref "0_overview" >}}).
 
-To see the full list, please go to [Quickstarts Overview]({{< ref "0_overview" >}}).
+This first quickstart provides step by step instructions to set up
+IdentityServer in the most basic scenario: protecting APIs for server-to-server
+communication. You will create a solution containing three projects:
+- An Identity Server
+- An API that requires authentication
+- A client that accesses that API
 
-This first quickstart is the most basic scenario: using Duende IdentityServer for protecting APIs for server-to-server communication.
-
-In this quickstart you define an API and a Client with which to access it. 
-The client will request an access token from the Identity Server using its client ID and secret and then use the token to gain access to the API.
+The client will request an access token from IdentityServer using its
+client ID and secret and then use the token to gain access to the API.
 
 ## Source Code
-As with all of these quickstarts you can find the source code for it in the [docs]({{< param qs_base >}}) repository. 
-The project for this quickstart is [Quickstart #1: Securing an API using Client Credentials]({{< param qs_base >}}/1_ClientCredentials).
+As with all of the quickstarts, you can find the source code for it in the
+[docs]({{< param qs_base >}}) repository. The project for this quickstart is
+[Quickstart #1: Securing an API using Client Credentials]({{< param qs_base >}}/1_ClientCredentials).
 
 ## Preparation
-The Duende IdentityServer templates for the dotnet CLI are a good starting point for the quickstarts.
-
-To install the templates open a console window and type the following command:
+The IdentityServer templates for the dotnet CLI are a good starting point for
+the quickstarts. To install the templates open a console window and type the
+following command:
 
-```
+```console
 dotnet new --install Duende.IdentityServer.Templates
 ```
 
-They will be used as a starting point for the various tutorials.
+## Create the Solution and IdentityServer Project
+In this section, you will create a directory for the solution and use the
+isempty (IdentityServer Empty) template to create an ASP.NET Core application
```

Comment:
Maybe makes sense to use italics on the "isempty" keyword. We try to do this elsewhere for all keywords (file names, C# type names, etc).